_tools/fpki-stat-report

```diff
@@ -0,0 +1,113 @@
+# The Federal PKI Management Authority Statisical Report
```

Comment:
Needs to be a markdown file (.md)
Needs the header info

---
layout: default 
title: 
collection: tools
permalink: tools/<linklocation>/
---

Reply:
Updated.